anime_downloader/config.py

```diff
@@ -82,6 +82,9 @@
         },
         'animedaisuki': {
             "servers": ["official"]
+        },
+        'vidstream': {
+            "servers": ["vidstream","gcloud","mp4upload","cloud9","hydrax"]
```

Comment:
I don't like. Why are these needed?

Reply:
So you could choose which vidstream server you prefer, they're different in terms of download speeds (and probably sizes + quality)

Reply:
You should add this in the docs somewhere and it needs doc comments in the vidstream class.